### ISSUE TITLE
ctex: 在最新版 LuaLaTeX 下编译错误

### ISSUE THREAD

**Issue by qinglee** [CTeX-org/ctex-kit#497]
ctex v2.4.16 在最新的 TeX Live 2020 下编译会报错：

```
) (c:/texlive/2020/texmf-dist/tex/xelatex/xecjk/xunicode-extra.def)...exlive/20
20/texmf-dist/tex/luatex/luatexja/ltj-jfont.lua:473: table index is nil
stack traceback:
        ...exlive/2020/texmf-dist/tex/luatex/luatexja/ltj-jfont.lua:473: in field 'add
_kyenc_list'
        [\directlua]:1: in main chunk.
<to be read again>
\cs_new_protected_nopar:Npn
l.319 \cs_new_protected_nopar:Npn
                                \__ctex_ltj_change_encoding:
?
```

原因是 luatexja 20200412.0 开始采用 `\luadef` 和 `token.scan_argument` 处理 TeX 与 lua 脚本的交互问题，改动了部分 API。

https://osdn.net/projects/luatex-ja/scm/git/luatexja/commits/b63e8444e3a6c4bad1f38e73d1ed8e99751ee85d

**Comment by chrisyeh96** [comment]
I am running into the same error using `ctex` package v2.4.16. I see that commit 5ce1dfa supposedly fixes this issue for the upcoming v2.5 release. Will v2.5 be released soon (within the next 2 days)?

Otherwise, are there any good workarounds for this problem?

**Comment by Liam0205** [comment]
@chrisyeh96 You can download and compile the in-developing version of CTeX 2.5, and put the compiled files under your TEXMFLOCAL TDS. After these, run texhash to rebuild the ls-R database to use the new version.
